apps/web/src/components/Footer/index.tsx

```diff
@@ -0,0 +1,50 @@
+import styled from 'styled-components'
+import { GithubLogo } from '../Icons/Github'
+import { TwitterXLogo } from '../Icons/TwitterX'
+import { DiscordLogo } from '../Icons/Discord'
+import { LinkedInLogo } from '../Icons/LinkedIn'
+import { HemiLogo } from '../Logo/HemiLogo'
+
+const StyledFooter = styled.div`
```

Comment:
should it be `footer` ?
```suggestion
const StyledFooter = styled.footer`
```